.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -9,7 +9,7 @@ assignees: ''
 
 ## 🚨🚨 Feature Request
 
-- [ ] Related to an existing [Issue](../../../issues) 
+- [ ] Related to an existing [Issue](../issues) 
```

Comment:
Using the relative path like `../issues` or `../../issues` depends on how the current URL (when making the issue) looks like (if the branch is not master/default then there's an extra "blob") so to be on the safer side, I think we should go with absolute path i.e this `https://github.com/activeloopai/Hub/issues` (should work in all cases)